cmd/adapter/adapter.go

```diff
@@ -406,3 +410,16 @@ func makePrometheusCAClient(caFilePath string, tlsCertFilePath string, tlsKeyFil
 		},
 	}, nil
 }
+
+func parseHeaderArgs(args []string) map[string][]string {
+	headers := make(map[string][]string, len(args))
+	for _, h := range args {
+		parts := strings.SplitN(h, "=", 2)
+		value := ""
+		if len(parts) > 1 {
```

Comment:
else sounds like a misconfiguration that we should make users aware of by returning an error here

Reply:
Thought about that but it looks like headers can have no value, e.g: https://github.com/whatwg/fetch/issues/332 - probably not useful in this context though so open either way.
So, should I return an error?

Reply:
Oh, good catch! I double-checked the RFC and empty headers are indeed valid. Since that's the case, I am fine with not erroring here.